src/hooks/useRoleByUserIds.js

```diff
@@ -1,26 +1,23 @@
-import { useNamespace, useOkapiKy } from '@folio/stripes/core';
-import { useQuery } from 'react-query';
+import { useChunkedCQLFetch } from '@folio/stripes/core';
 
 function useRoleByUserIds(users) {
```

Comment:
It took me a while to figure out what this hook does, and what the underlying data structure is (we're querying a join-table here with one-half of the PK (the userId, the other half being the roleId). Documentation would be really helpful to make this a bit less opaque.

Reply:
Done